pipelines/coaddQAPlots.yaml

```diff
@@ -260,3 +260,18 @@ tasks:
         from lsst.analysis.drp.dataSelectors import (CoaddPlotFlagSelector, FlagSelector, GalaxyIdentifier,
         SnSelector, UnknownIdentifier)
         from lsst.analysis.drp.calcFunctors import MagDiff
+
+  plot_CircAp9_forced_skyObj_sky:
+    # Make a plot showing the 9-pixel radius circular aperture flux in units of nJy.
+    # This plot shows the on sky distribution of these values for sky objects.
+    class: lsst.analysis.drp.skyPlot.SkyPlotTask
+    config:
+      axisActions.zAction.column: "iAp9Flux"
```

Comment:
I see there is precedent from the gen2 version of this plot, but why use a 9-pixel aperture when all the others are 12? Is it too hard to find empty apertures if you use 12?

Reply:
Right, the 9-pixel aperture is the closest we have to an aperture that matches the "radius" of sky objects. Currently, sky objects are assigned to empty patches of sky where no other detected footprint exists within an 8-pixel radius. In fact, DM-23779 is tasked with increasing this from 8 to 9 pixels, to allow for our 9 pixel aperture to exactly match sky object apertures in future analyses.